src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -919,6 +929,16 @@ private static CompletionItem CreateResourceOrModuleConditionCompletion(Range re
                 .Build();
         }
 
+        private static CompletionItem CreateConditionCompletion(Range replacementRange)
+        {
+            const string conditionLabel = "if-else";
+            return CompletionItemBuilder.Create(CompletionItemKind.Snippet, conditionLabel)
+                .WithSnippetEdit(replacementRange, "${1:condition} ? ${2:TrueValue} : ${3:FalseValue}")
```

Comment:
>FalseValue [](http://example.com/codeflow?start=90&length=10)

Btw, where does the cursor land if you tab through all the snippet placeholders? Does it land at the end?

Reply:
Discussed offline. Cursor lands at the end.